sw/airborne/modules/ctrl/vertical_ctrl_module_demo.c

```diff
@@ -48,7 +48,7 @@ PRINT_CONFIG_VAR(VERTICAL_CTRL_MODULE_AGL_ID)
 static abi_event agl_ev; ///< The altitude ABI event
 
 /// Callback function of the ground altitude
-static void vertical_ctrl_agl_cb(uint8_t sender_id __attribute__((unused)), float distance);
+static void vertical_ctrl_agl_cb(uint8_t sender_id, uint32_t stamp, float distance);
```

Comment:
missing `unused` attribute

Reply:
This is only needed in the real implementation not the definition.